config/rootfs/chromiumos/scripts/build_board.sh

```diff
@@ -25,6 +25,12 @@ function cleanup()
 
 trap cleanup EXIT
 
+if [ "$2" != "release-R114-15437.B" ] || [ -n "${VANILLA_MANIFEST}" ]; then
```

Comment:
It is actually good point, i am cooking better idea how to implement this, and something that @mgalka had in mind - to reduce complexity of this shell file, we can split it, and make version specific workarounds as separate shell script, and just verify if it does exist.
This will solve even more issues - we can keep old versions buildable by keeping old workaround files, and if file doesn't exist for new version, it will serve as reminder to verify and probably "port" existing workarounds.